<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QBCfgTabPageUserGeneralUi</class>
 <widget class="QWidget" name="QBCfgTabPageUserGeneralUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>General</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="Q3GroupBox" name="groupBox6">
     <property name="title">
      <string>User Settings</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="1">
       <widget class="QLineEdit" name="userNameEdit"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="textLabel1_2">
        <property name="text">
         <string>User Name</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="userIdLabel">
        <property name="text">
         <string>User Id</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="userIdEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="customerIdLabel">
        <property name="text">
         <string>Customer Id</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="customerIdEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Q3GroupBox" name="groupBox7">
     <property name="title">
      <string>Bank Settings</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="1">
       <widget class="QComboBox" name="countryCombo"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="textLabel1_4">
        <property name="text">
         <string>Bank Id</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="textLabel2">
        <property name="text">
         <string>Country</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLineEdit" name="bankIdEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="bankIdButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>userNameEdit</tabstop>
  <tabstop>userIdEdit</tabstop>
  <tabstop>customerIdEdit</tabstop>
  <tabstop>countryCombo</tabstop>
  <tabstop>bankIdEdit</tabstop>
  <tabstop>bankIdButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
